Models/Training.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Training
```

Comment:
This type should be `TrainingProgram`

Reply:
Spoke with Steve on the naming convention of Training. We identified that their is a table in our DB by the name of "Trainings" and are okay to proceed with the Training model.